apps/routecraft.dev/src/app/docs/reference/operations/concurrency/page.md

```
@@ -0,0 +1,117 @@
+```ts
+craft()
+  .id('reserve-inventory')
+  .from(source)
+  .concurrency({ max: 5 }) // at most 5 reservations in flight at once
+  .to(http({ url: 'https://inventory.internal/reserve' })) // bounded
+  .transform(formatReceipt) // NOT bounded
+```
+
+**Mental model:** A pool of `max` slots. An exchange takes a slot before the wrapped work and frees it the moment the work settles (success, drop, or failure). When every slot is busy:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)